CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ A brief description of the categories of changes:
   ([617](https://github.com/bazelbuild/rules_python/issues/617)).
 * (pypi) When {attr}`pip.parse.experimental_index_url` is set, we need to still
   pass the `extra_pip_args` value when building an `sdist`.
+* (pypi) The patched wheel filenames from now on are using local version specifiers
```

Comment:
```suggestion
* (pypi) The patched wheel filenames from now on are using local version identifiers
```

Alternative suggestion for full CHANGELOG entry:
`(pypi) Patched wheel filenames use local version identifiers to enable installation with standards-compliant package managers`